moving-forward-series/simple-payment-verification.md

```diff
@@ -0,0 +1,31 @@
+
+## Decred’s Implementation of SPV
+When Decred implemented the “Block Header Commitments” consensus upgrade in 2020 (DCP: 0005 “Block Header Commitments”) it focused on mitigating against these attack vectors to bring the SPV security level closer to that of a fully validating node. This was done through a combination of committed fraud proofs, inclusion proofs, and filters. SPV nodes on the Decred network can now quickly verify associated transactions and securely reject the invalid ones accordingly. 
+
+## Lite clients vs SPV
+SPV and Fully validating nodes are my wallets of choice, as they offer the highest levels of security whilst verifying blockchain data independently. But, of course, there are other solutions, commonly referred to as lite clients. A large majority of multi-coin wallets or exchange wallets favour lite clients or fully custodial wallets. This is mainly due to improving the user experience and making the functionality more streamlined by bypassing the verification and blockchain download process.
+
+A typical setup for a lite client wallet is to connect directly to a centralised server controlled, normally, by the wallet provider. Your wallet then uses the centralised server’s API to request information, like, what’s my balance, or what are my transactions. Which means you have to trust the information provided by the server in the hope that the server isn’t lying or providing inaccurate information.
+
+There are a multitude of issues that could arise from this setup, including, what happens to your wallet and coins if a provider closes down? In essence, these wallet providers are the gateway to your funds, similar to that of a custodial service. Is it even possible to get your funds if the service is no longer available?
+
+## Interchangeable SPV Wallets
+Another interesting thing about Decred’s implementation of SPV is it was built as part of the core software “dcrwallet CLI tool”, and as such, can be used on any wallet that correctly implements it. Current implementations include dcrwallet, Decrediton and CryptoPower. As an added bonus, Decred’s SPV mode is also interchangeable with the fully validating wallet if the need arises to increase your wallets security or add additional blockchain functionality.
+
+## Benefits and drawbacks
+To conclude, SPV and Lite clients manage risk and functionality in various ways. If you require the highest assurances, there is still no better option than running a fully validating wallet. But if you’re seeking a lightweight alternative to interact with the blockchain, SPV is a pretty solid option that has seen many improvements over the last few years to increase security and privacy. 
+
+As it currently stands, the main drawbacks with using lite clients and custodial wallets over a properly implemented SPV system. Revolve around the ability to validate the data the wallet receives and the need to trust third-party services or servers to process the information required. 
```

Comment:
These two sentences are not full sentences, and need to be combined into one sentence. However, when you do that, it becomes a very long run-on sentence. Maybe just cut out the fluff and shorten it to:

* "As it currently stands, the main drawbacks with using lite clients revolve around having to trust centralized third-party services instead of the decentralized network."